framework_tvb/tvb/adapters/simulator/simulator_fragments.py

```diff
@@ -264,6 +265,9 @@ def __init__(self, pse_param1, pse_param2):
         if pse_param2:
             self._add_pse_field(pse_param2, self.KEY_PARAM2)
 
+        self.max_pse_number = HiddenField(Int(default=TvbProfile.current.MAX_RANGE_NUMBER, required=False),
```

Comment:
Isn't this part of another PR?

Reply:
A rebase and forced push solved the issue.